README.md

```diff
@@ -10,9 +10,9 @@ Shipyard is an Entity Component System focused on usability and speed.
 If you have any question or want to follow the development more closely join the [Zulip](https://shipyard.zulipchat.com).
 
 There's two big learning resources:
-- (Soon™) The Tutorial for people new to ECS or preferring to learn by making a project.
+- (Soon™) The Tutorial for people new to ECS or who prefer to learn by making things.
 - [The Guide](https://leudz.github.io/shipyard/guide) for people that already know how to use an ECS and mostly want to learn Shipyard's syntax.  
-  It also go more in depth and provide useful recipes for common patterns.
+  It also goes into greater depth and provides useful recipes for common design patterns.
```

Comment:
Currently there are three recipes: a migration guide, a design pattern and an example of how to use shipyard with seed. Maybe it's best to not specify what they are for.
```suggestion
  It also goes into greater depth and provides useful recipes.
```